static/quickstart.js

```diff
@@ -110,7 +65,7 @@
   };
 
   speakerDevices.addEventListener("change", function () {
-    var selectedDevices = [].slice
+    const selectedDevices = [].slice
```

Comment:
Looks like this got dredged up by the var -> const change, haha.

`[].slice.call` is pretty esoteric way to convert a `NodeList` into an `Array`. Would recommend using the more readable/beginner-friendly `Array.from` method. [MDN link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#array_from_a_nodelist)

```suggestion
    const selectedDevices = Array.from(speakerDevices.children)
      .filter((node) = > node.selected); // the filter function can also be shorted :)
```